<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DUdpDriv</class>
 <widget class="QDialog" name="DUdpDriv">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>500</y>
     <width>201</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="ui_send_txt">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>520</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="ui_recv_txt">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>480</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ui_send_act">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>520</y>
     <width>89</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ui_send_path">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>301</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>D:\e1_library\f2_light_eye_plat2\light_eye_pc_sys\light_eye_v02_4.dat\demo.bmp</string>
   </property>
  </widget>
  <widget class="QLabel" name="ui_send_img">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>281</width>
     <height>301</height>
    </rect>
   </property>
   <property name="text">
    <string>图片发送</string>
   </property>
  </widget>
  <widget class="QLabel" name="ui_recv_img">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>120</y>
     <width>341</width>
     <height>301</height>
    </rect>
   </property>
   <property name="text">
    <string>图片接收</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ui_img_act">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>89</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>发送图片</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ui_recv_path">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>80</y>
     <width>301</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>D:\e1_library\f2_light_eye_plat2\light_eye_pc_sys\light_eye_v02_4.dat\demo3.bmp</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DUdpDriv</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DUdpDriv</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
